app/models/ChannelType.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+sealed trait ChannelType
+
+object ChannelType extends Enumerable.Implicits {
+  case object web  extends ChannelType
+  case object api  extends ChannelType
+  case object ncts extends ChannelType
```

Comment:
We need to confirm that we need to add "ncts" for the channel. Should this not only be web/api?

Reply:
at the moment its because audit-wise its behaving as an origin-tracker, but I appreciate that in the departure model the channel will be used for filtering. So when that works happens ncts may well be removed and instead source the channel of the departure directly from that route. But at the moment there is no way to discern the correct channel.

Reply:
@amaalali I've spoken with Maria Decosta about this and she's of the opinion that ncts doesn't add value (which I agree with). I was going to remove this as part of the work to only provide web or api departures/arrivals to web/api, are you happy with that or would you rather I remove nct and put in a different default value for now?